### ISSUE TITLE
RQLITE_HISTFILESIZE=0 still writes an .rqlite_history file

### ISSUE THREAD

**Issue by jamielinux** [rqlite/rqlite#1145]
**What version are you running?**

```
Version v7.13.0, commit f4a1583d3a63e55ddc7d310c64d8a6cd0a0f2b1d, branch master, built on 2022-12-15T11:04:54-0500
```

**Are you using Docker or Kubernetes to run your system?**

No.

**Are you running a single node or a cluster?**

3-node cluster.

**What did you do?**

Set `RQLITE_HISTFILESIZE=0` in the shell environment.

**What did you expect to happen?**

No `.rqlite_history` file created.

**What happened instead?**

An empty (zero-byte) `.rqlite_history` file was created.

Obviously this isn't a massive issue, but I think it'd be better if it doesn't create an empty file when you set the size to zero.

Thank you!

**Comment by otoolep** [comment]
Thanks for the report @jamielinux

**Comment by jamielinux** [comment]
Thank you @otoolep for already creating a PR!

**Comment by otoolep** [comment]
Fixed in 7.13.1.
